Essentials/src/messages_cs.properties

```diff
@@ -507,6 +507,7 @@ teleportationEnabledFor=\u00a76Teleportace \u00a7apovolena\u00a76 hraci \u00a7c{
 teleportAtoB=\u00a76Hrac \u00a7c{0} \u00a76te teleportoval k hraci \u00a7c{1}\u00a76.
 teleportDisabled=\u00a7c{0} \u00a74mas teleportaci zablokovanou.
 teleportHereRequest=\u00a76Hrac \u00a7c{0}\u00a76 te pozadal aby si se teleportoval/a k nemu.
+teleportHome=\u00a76Teleporting to {0}.
```

Comment:
Due to th Crowdin integration, these need removing from each translated messages file - added messages should now be in `messages.properties`, and Crowdin will handle the rest.